webpack/scenes/ContentViews/Table/tableDataGenerator.js

```diff
@@ -1,137 +1,86 @@
-    { title: __('Details'), props: { isOpen: false, ariaControls: `cv-details-expansion-${id}` } },
-    {
-      title: <IconWithCount Icon={ScreenIcon} count={environments.length} title={`environments-icon-${id}`} />,
-      props: { isOpen: false, ariaControls: `cv-environments-expansion-${id}` },
-    },
+    { title: lastPublished ? <LongDateTime date={lastPublished} showRelativeTimeTooltip /> : <InactiveText text="Not yet published" /> },
+    { title: <LastSync lastSync={lastTask} lastSyncWords={lastSyncWords} emptyMessage="N/A" /> },
     {
-      title: <IconWithCount Icon={ContainerNodeIcon} count={versions.length} title={`versions-icon-${id}`} />,
-      props: { isOpen: false, ariaControls: `cv-versions-expansion-${id}` },
+      title: latestVersion ? <ContentViewVersionCell {...{
+ id, latestVersion, latestVersionId, latestVersionEnvironments,
+}}
+      /> : <InactiveText text="Not yet published" />,
     },
   ];
-  if (openColumn) row[openColumn].props.isOpen = true;
   return row;
 };
 
-const buildDetailDropdowns = (id, rowIndex, openColumn) => {
-  const cvId = { cvId: id };
-  const expansionProps = { ...cvId, className: 'pf-m-no-padding' };
-  const containerProps = column => ({ ...cvId, isOpen: openColumn === column });
-  const offsetColumn = 3; // index of first expandable column
+const buildExpandableRows = (contentViews) => {
+  const rows = [];
+  let cvCount = 0;
 
-  let detailDropdowns = [
-    {
-      compoundParent: offsetColumn,
-      cells: [
-        {
-          title: (
-            <DetailsContainer {...containerProps(offsetColumn + 1)}>
-              <DetailsExpansion {...expansionProps} />
-            </DetailsContainer>),
-          props: { colSpan: 6 },
-        },
-      ],
-    },
-    {
-      compoundParent: offsetColumn + 1,
+  contentViews.forEach((contentView) => {
+    const {
+      id, name, description, activation_keys: activationKeys, hosts,
+    } = contentView;
+    const cells = buildRow(contentView);
+    const cellParent = {
+      cvId: id, cvName: name, isOpen: false, cells,
+    };
+    rows.push(cellParent);
+    const cellChild = {
+      parent: cvCount,
       cells: [
         {
-          title: (
-            <DetailsContainer {...containerProps(offsetColumn + 2)}>
-              <EnvironmentsExpansion {...expansionProps} />
-            </DetailsContainer>),
-          props: { colSpan: 6 },
+          title: <DetailsExpansion cvId={id} {...{ activationKeys, hosts }} />,
+          props: {
+            colSpan: 2,
+          },
         },
-      ],
-    },
-    {
-      compoundParent: offsetColumn + 2,
-      cells: [
         {
-          title: (
-            <DetailsContainer {...containerProps(offsetColumn + 3)}>
-              <VersionsExpansion {...expansionProps} />
-            </DetailsContainer>),
-          props: { colSpan: 6 },
+          title: description ? <>{__('Description: ')}{__(description)}</> : <InactiveText text="No Description" />,
```

Comment:
`__(description)` shouldn't be translated, since `__` should only be used on literal strings.  (`__('Description: ')` is fine though, to be clear)

Reply:
What are your thoughts on displaying the description without the word "Description"?  I know it's in the mockup, but it seems unnecessary to me. 
```suggestion
          title: description ? <>{description}</> : <InactiveText text={__("No description")} />,
```
(also remember not to capitalize the second word :)

Reply:
I don't mind removing it..The description text should be self explanatory..